src/main/webapp/entries.js

```diff
@@ -86,4 +86,5 @@ module.exports = {
   "user-details": "./resources/js/pages/user-details.js",
   "project-metadata-templates":
     "./resources/js/pages/projects/metadata-templates",
+  admin: "./resources/js/pages/admin/index.js",
```

Comment:
FYI, and please don't bother changing it this is just a teaching point.  You don't have to include `index.js`, that is the default file name for js files so if you just had `admin: "./resources/js/pages/admin"` it would also be valid.